tests/Klasse.spec.ts

```diff
@@ -44,117 +51,280 @@ test.describe(`Testfälle für die Administration von Klassen: Umgebung: ${proce
     });
   });
 
-  test("Eine Klasse als Landesadmin anlegen und die Klasse anschließend in der Ergebnisliste suchen und dann löschen", {tag: [LONG, SHORT, STAGE]}, async ({ page }) => {
-    const startseite: StartPage = new StartPage(page);
+  test(
+    'Eine Klasse als Landesadmin anlegen und die Klasse anschließend in der Ergebnisliste suchen und dann löschen',
+    { tag: [LONG, SHORT, STAGE] },
+    async ({ page }) => {
+      const startseite: StartPage = new StartPage(page);
+      const menue = new MenuPage(page);
+      const klasseCreationView = new KlasseCreationViewPage(page);
+      const klasseManagementView = new KlasseManagementViewPage(page);
```

Comment:
Bitte dies hier löschen und im nächsten Step mit `FromAnywhere(page).start()` anfangen und von dort aus die jeweiligen Methoden nutzen, um die gewünschte Seite klasseCreationView zu returnen. Beispiel in Import.spec.ts ab Z31.